### DISCUSSION TITLE
PhotoMod?

### DISCUSSION THREAD

**Discussion by ZeligitFS** [codecrafting-io/AdvancedPathTracingCP2077#2] [General]
Have a nice day! Should this mod work if I have path tracing enabled **only** for photo mode? I installed it and all the necessary components, changed modes and settings in **ReSTIR DI** mode. BUT! I don't see any difference in the quality of frames in photo mode. Also, their processing time doesn't change.

I can't test its functionality in the main game, because my video card is quite weak and can't handle path tracing.

**Comment by codecrafting-io** [comment]
Hi, unfortunately PT photo mode only is not fully supported at the moment, but I can look if it's possible. I think it is, I just have to reapply the settings when entering the photo mode, but I can't guarantee that it will work properly for now. Stay tuned for more updates in January. Thanks!

**Reply by ZeligitFS** [reply]
Thanks for the answer! I'll keep an eye on you)

**Reply by theShadowreaper** [reply]
There used to be a mod that had settings specifically for photo mode but it was deleted from nexusmods quite a while ago (called Preem Photos I believe)
that mod changed these ini settings:
RayTracing/Reference/BounceNumberScreenshot
RayTracing/Reference/RayNumberScreenshot
RayTracing/ReferenceScreenshot/SampleNumber

saw this post and thought I could maybe point you in the right direction :)
also slight personal stake in this because with update 2.2 the mod I'm taking this from bricks photo mode ;-;

I recommend maybe looking into these two as well since people will most likely crank the settings in photo mode:
Rendering/FakeGPUVRAM
Rendering/FakeOverrideGPUVRAM
(BUT I don't know what side effects these may have)

**Comment by codecrafting-io** [comment]
@theShadowreaper @ZeligitFS Just released [**v0.6.0**!](https://github.com/codecrafting-io/AdvancedPathTracingCP2077/releases). Just to be clear, the added experimental support is for Photo Mode screenshots, not Photo Mode itself. You can leave the path tracing mode on any option, but for screenshots the photo mode switches to ReSTIR DI and should use the rays and bounces defined in the settings

**Reply by ZeligitFS** [reply]
Hi, it works very well in photo mode. Thank you so much for your work)!
